frontend/src/components/NavBar.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import {
+  Button,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  Nav,
+  Navbar,
+  NavbarBrand,
+  NavItem,
+  UncontrolledDropdown
+} from 'reactstrap'
+import LanguageSelector from './LanguageSelector'
+import {withRouter} from 'react-router-dom';
+import k_logo from "../media/greenK.png"
+import kiva_logo from '../media/kivaPlainLogo.png'
+import info_image from '../media/gray_info.png'
+import sandwich_image from '../media/sandwich.png'
+import '../styles/navbar.css'
+
+const NavBarComponent = withRouter(props => <NavBar {...props}/>);
+
+class NavBar extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isLoginPage: true
+    }
+
+    this.handlePageChange = this.handlePageChange.bind(this)
+  }
+
+  handlePageChange() {
+    if (this.props.location.pathname.equals("http://localhost:3000/dashboard")) {
```

Comment:
I don't believe that string comparison is done with .equals() in Javascript, as far as I know it's done using ===, but I could be wrong on that

Reply:
Also double check the value of this.props.location.pathname, it might actually exclude localhost:3000 and everything before that and only include the page route, so something like "/" or "/dashboard"